server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -100,131 +101,115 @@ public void teardown() {
         jmapServer.stop();
     }
     
-    @Test
```

Comment:
I don't understand test you had seems to me to be for totally different thinks than test you delete.
So if I am correct why do you delete those old test ?